Automattic/wp-calypso
# DIFM Website Content: Fetch data from new endpoint

#### Proposed Changes

* Fetches DIFM data from the new endpoint added in D91173-code instead of the get site endpoint.

#### Testing Instructions



* Apply D91173-code
* Go to `/start/do-it-for-me` and complete the purchase.
* After purchase, confirm that you are taken to the website content form, and that the pages you selected earlier in the flow are available.

#### Pre-merge Checklist
